tests/DbaaS/pages/dbaasPage.js

```diff
@@ -88,6 +88,9 @@ module.exports = {
           nodesFieldErrorMessage: '$nodes-field-error-message',
           nodesNumberField: '$nodes-number-input',
           resourcesPerNode: (clusterSize) => `//label[contains(text(), "${clusterSize}")]`,
+          dbClusterExternalAccessTooltip: locate('$expose-field-container').find('.css-sr6nr'),
+          dbClusterExternalAccessCheckbox: '$expose-switch',
+          dbClusterExternalAccessTooltipText: locate('div').withAttr({ class: 'popper' }),
```

Comment:
this class is correct?

Reply:
I think it is, the test passed https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/129/pipeline.